engines/bops_config/app/views/bops_config/application_types/features/edit.html.erb

```diff
@@ -0,0 +1,64 @@
+<% content_for :page_title do %>
+  <%= t(".choose_features") %> - <%= t("page_title") %>
+<% end %>
+
+<% add_parent_breadcrumb_link "Home", root_path %>
+<% add_parent_breadcrumb_link "Application Types", application_types_path %>
+
+<% content_for :title, t(".choose_features") %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-l"><%= t(".choose_features") %></h1>
+    <h2 class="govuk-heading-m"><%= @application_type.description %></h2>
+
+    <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+
+    <%= form_with model: @application_type, url: [@application_type, :features] do |form| %>
+      <%= form.govuk_error_summary(presenter: ErrorPresenter) %>
+
+      <%= form.fields_for :features, @application_type.features do |ff| %>
+        <div class="govuk-form-group">
+          <fieldset class="govuk-fieldset">
+            <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+              <%= t("legends.check_application_details", scope: :application_type_features) %>
+            </legend>
+
+            <div class="govuk-checkboxes" data-module="govuk-checkboxes">
+              <div class="govuk-checkboxes__item">
+                <%= ff.check_box :planning_conditions, class: "govuk-checkboxes__input" %>
+                <%= ff.label :planning_conditions, t("labels.planning_conditions", scope: :application_type_features), class: "govuk-label govuk-checkboxes__label" %>
+              </div>
+
+              <div class="govuk-checkboxes__item">
+                <%= ff.check_box :permitted_development_rights, class: "govuk-checkboxes__input" %>
+                <%= ff.label :permitted_development_rights, t("labels.permitted_development_rights", scope: :application_type_features), class: "govuk-label govuk-checkboxes__label" %>
+              </div>
+            </div>
+          </fieldset>
```

Comment:
Seems like we're generating a lot of manual HTML here - there should be a better way, either using `govuk_collection_check_boxes` or a combination of  `govuk_check_boxes_fieldset` and `govuk_check_box`. There's some decent examples in the [form builder guides][1].

[1]: https://govuk-form-builder.netlify.app/form-elements/checkboxes/